### ISSUE TITLE
Enable pane focus follows mouse.

### ISSUE THREAD

**Issue by caedmon-kitty** [zellij-org/zellij#4361]
I am not sure if this is implemented already but I would like the pane focus to follow the mouse.  When I move the cursor to a pane that pane should have keyboard focus without hitting any key or a mouse button.  I have my entire desktop setup this way.  Having to click on a pane or use the keyboard to shift focus disturbs my workflow.

**Comment by cyrusduong** [comment]
Yes, would love for this to be an option for those that are familiar/use tiling window managers. This is a common option.

**Comment by non-bin** [comment]
bump

**Comment by robertoash** [comment]
[This PR](https://github.com/zellij-org/zellij/pull/4364) has been out since August?

I hope it gets merged soon!

**Comment by tjhanley** [comment]
bump

**Comment by coexactly** [comment]
I would also love this

**Comment by pumbaa666** [comment]
Same

**Comment by pumbaa666** [comment]
There is also a 4 years old feature request here [#1679](https://github.com/zellij-org/zellij/issues/1679)

**Comment by imsnif** [comment]
Been working for a while behind a config.
